project/ProjectPlugin.scala

```diff
@@ -92,24 +90,15 @@ object ProjectPlugin extends AutoPlugin {
 
   }
 
-  import autoImport._
-
   override def projectSettings: Seq[Def.Setting[_]] =
     Seq(
-      name := "evaluator",
-      description := "Scala Exercises: The path to enlightenment",
-      organization := "org.scala-exercises",
-      organizationName := "47 Degrees",
-      organizationHomepage := Some(url("https://47deg.com")),
       scmInfo := Some(
         ScmInfo(
           url("https://github.com/scala-exercises/evaluator"),
           "scm:git:https://github.com/scala-exercises/evaluator.git",
           Some("scm:git:git@github.com:scala-exercises/evaluator.git")
         )
       ),
```

Comment:
This is already provided by [sbt-ci-release](https://github.com/olafurpg/sbt-ci-release/blob/7fcf8c79e891d3c9793264fc8b5ea78070684ed5/plugin/src/main/scala/com/geirsson/CiReleasePlugin.scala#L96-L115)

Reply:
Yeah, but last time if failed the heroku deployment, that's why they needed to be added

Reply:
![](https://media3.giphy.com/media/14tIO0uZ8UFG3C/giphy.gif?cid=6104955e12dd0f00fa5e3939588c4c28bd96b6608cc78818&rid=giphy.gif)